src/Main.elm

```diff
@@ -69,17 +41,15 @@ arrivalBarGraph : Spec
 arrivalBarGraph =
     let
         des =
-            description "A simple bar chart with embedded data."
+            description "A visualization of arrival time averages for the MTA G Train Greenpoint Ave. stop"
 
         data =
-            dataFromColumns []
-                << dataColumn xAxisName (strs fakeArrivalTimes.day)
-                << dataColumn yAxisName (nums fakeArrivalTimes.meanArrivalDifferences)
+            dataFromUrl "http://localhost:4000/" []
```

Comment:
Is it going to be retrieving from `localhost` when deployed? Or should this be an env var?

Reply:
This is a bit tricky in elm. I'll have to do more research and add it in the next iteration